packages/core/src/components/ButtonGroup/ButtonGroup.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import { Meta, StoryObj } from '@storybook/react';
+import { css, useTheme } from '@emotion/react';
+import { ButtonGroup } from './ButtonGroup';
+
+const items = [
```

Comment:
I think, we could move out items const to the external file? We use it twice

Reply:
done